### DISCUSSION TITLE
Android and iOS: Is there an SDK for a "pop over context menu"?

### DISCUSSION THREAD

**Discussion by FizixTiramu** [appcelerator/titanium_mobile#14025] [General]
I'm not sure what the correct terminology is for this, so I've attached an image thats almost identical to what my client is looking for. 

Basically, a popover context menu that appears when you long-tap on content that reveals options with an icon and label for various tasks such as "translate", "transliterate", "share", "copy". The type that hovers over the content with an arrow and disappears when you tap away from it.

Is there a built in SDK for this? or a module?

Most apps that have them seem consistent so I'm thinking there is a native way and I'd prefer to stick to that than custom build one (I can see a lot of events spewing out from it if I do). I've dug through the docs and I'm not seeing anything, not knowing the proper terminology for these probably isn't helping. :D

![custom_pop_up_menu](https://github.com/tidev/titanium-sdk/assets/156670741/9a5fbff6-d562-42af-ba09-d2d6de34f6ce)

**Comment by m1ga** [comment]
There is https://github.com/hansemannn/titanium-tooltip but I've not used it. For iPad there is a native popover in the SDK https://titaniumsdk.com/api/titanium/ui/ipad/popover.html
The first link has an Android version too but I also have this one: https://github.com/m1ga/ti.popover but not updated that in a while :smile: Should still work I hope

**Comment by FizixTiramu** [comment]
Thanks, I'll check those out
